crates/rust-analyzer/Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
 name = "rust-analyzer"
 version = "0.0.0"
-description = "TBD"
+authors = ["rust-analyzer Team"]
```

Comment:
This field has been optional since https://rust-lang.github.io/rfcs/3052-optional-authors-field.html.

> The crates.io registry does not allow users to change the contents of already published versions: this is highly desirable to ensure working builds don't break in the future, but it also has the unfortunate side-effect of preventing people from updating the list of crate authors defined in Cargo.toml's package.authors field.
>
> This is especially problematic when people change their name or want to remove their name from the Internet, and the crates.io team doesn't have any way to address that at the moment except for deleting the affected crates or versions altogether. We don't do that lightly, but there were a few cases where we were forced to do so.
>
> The contents of the field also tend to scale poorly as the size of a project grows, with projects either making the field useless by just stating "The $PROJECT developers" or only naming the original authors without mentioning other major contributors.

It was removed in https://github.com/rust-analyzer/rust-analyzer/pull/9493.

Reply:
I think it's ok to leave this in, if cargo-deb needs this. Although would be a good idea to patch cargo-deb to not require it either!

Reply:
Yeah, we should at least file an issue. Do you know why it fails without the filed? `cargo-deb` doesn't seem to use `cargo-metadata`.

Reply:
Ah, `cargo-deb` can also use `[package.metadata.deb]`, but I'm not sure that's better.